configs/mainnet-asset-white-list.json

```diff
@@ -5,414 +5,382 @@
     "decimal": 18,
-    "minimalBridgeAmount": "312000000000000000",
+    "minimalBridgeAmount": "652000000000000000",
     "bridgeFee": {
-      "in": "16600000000000000",
-      "out": "156000000000000000"
-    },
-    "sudtArgs": "0xaf47fbc88c924f720fd8d4c0aea5dca8bd00ceecd9f8c2ab45415f6a5dd2ee97"
+      "in": "36700000000000000",
+      "out": "326000000000000000"
+    }
   },
   {
     "address": "0x111111111117dC0aa78b770fA6A738034120C302",
     "name": "1INCH",
     "symbol": "1INCH",
     "logoURI": "https://cryptologos.cc/logos/1inch-1inch-logo.svg?v=013",
     "decimal": 18,
-    "minimalBridgeAmount": "33500000000000000000",
+    "minimalBridgeAmount": "66200000000000000000",
     "bridgeFee": {
-      "in": "1780000000000000000",
-      "out": "16700000000000000000"
-    },
-    "sudtArgs": "0x5bea7b77283ac8a20bd4e848a67c68349a3880b67278a5565bbbc242a26d3e29"
+      "in": "3720000000000000000",
+      "out": "33100000000000000000"
+    }
   },
   {
     "address": "0x3DB6Ba6ab6F95efed1a6E794caD492fAAabF294D",
     "name": "LTO",
     "symbol": "LTO",
     "logoURI": "https://cryptologos.cc/logos/lto-network-lto-logo.svg?v=002",
     "decimal": 8,
-    "minimalBridgeAmount": "29900000000",
+    "minimalBridgeAmount": "46300000000",
     "bridgeFee": {
-      "in": "1590000000",
-      "out": "14900000000"
-    },
-    "sudtArgs": "0x33e2afa67434322b1368bf5064b31dc20021bcccc871aa5cc6a5380aa58c59f6"
+      "in": "2600000000",
+      "out": "23100000000"
+    }
   },
   {
     "address": "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0",
     "name": "MATIC",
     "symbol": "MATIC",
     "logoURI": "https://cryptologos.cc/logos/matic-network-matic-logo.svg?v=002",
     "decimal": 18,
-    "minimalBridgeAmount": "81400000000000000000",
+    "minimalBridgeAmount": "67700000000000000000",
     "bridgeFee": {
-      "in": "4330000000000000000",
-      "out": "40700000000000000000"
-    },
-    "sudtArgs": "0xb571d967797059352785a0297b3ff740f17df1b97315c311b2c0e0107a0db5a8"
+      "in": "3810000000000000000",
+      "out": "33800000000000000000"
+    }
   },
   {
     "address": "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C",
     "name": "BNT",
     "symbol": "BNT",
     "logoURI": "https://cryptologos.cc/logos/bancor-bnt-logo.svg?v=002",
     "decimal": 18,
-    "minimalBridgeAmount": "25900000000000000000",
+    "minimalBridgeAmount": "47700000000000000000",
     "bridgeFee": {
-      "in": "1380000000000000000",
-      "out": "12900000000000000000"
-    },
-    "sudtArgs": "0x4192873b0a0aaeaa04372aa46ff2cd94a90ac1acaa7569458226febec4d6d5ed"
+      "in": "2690000000000000000",
+      "out": "23800000000000000000"
+    }
   },
   {
     "address": "0x56d811088235F11C8920698a204A5010a788f4b3",
     "name": "BZRX",
     "symbol": "BZRX",
     "logoURI": "https://cryptologos.cc/logos/bzx-protocol-bzrx-logo.svg?v=013",
     "decimal": 18,
-    "minimalBridgeAmount": "343000000000000000000",
+    "minimalBridgeAmount": "Inf00000",
```

Comment:
Seems not right here?

Reply:
The weightedAvgPrice returned from API is 0, I'll add a exception handle here.